### PR TITLE
fix: drop aggr tables in drop table

### LINKED ISSUE #3836: long_window bugs
# meta table and aggr table deletion
## workaround

```
select * from __INTERNAL_DB.PRE_AGG_META_INFO where base_db="db1" and base_table="t1";
-- 一般是删除这张表下所有的aggr表，假设这里查出来了的aggr_table只有一张__PRE_AGG_DB.pre_db1_d1_w1_count_，删表并删除其在meta表的信息
drop table __PRE_AGG_DB.pre_db1_d1_w1_count_;
delete from __INTERNAL_DB.PRE_AGG_META_INFO where aggr_table="pre_db1_d1_w1_count_";
```

## detail

P.S. long window definition can be w1:100(int literal), but if window agg is xx_where, it can't

### PR DESCRIPTION
**Pull request by vagetablechicken**
closes #3836

### CODE REVIEW

File: src/sdk/sql_cluster_router.cc [hunk 1]
```diff
@@ -875,54 +875,41 @@ bool SQLClusterRouter::DropTable(const std::string& db, const std::string& table
...
-        }
-        auto tablet_client = tablet_accessor->GetClient();
-        if (!tablet_client) {
-            SET_STATUS_AND_WARN(status, StatusCode::kCmdError, "get tablet client failed");
-            return false;
+            if (!DropTable(aggr_db, aggr_table, true, status)) {
+                WARN_NOT_OK_AND_RET(status, absl::StrCat("drop aggr table ", aggr_db, ".", aggr_table, " failed"),
+                                    false);
+            }
+            aggr_tables.emplace_back(aggr_db, aggr_table);
```
Comment (vagetablechicken) [inline src/sdk/sql_cluster_router.cc]: for:
  drop table
  delete from meta info

**Comment by vagetablechicken** [comment]
truncate base table, will aggr table be cleaned?

**Comment by vagetablechicken** [comment]
> truncate base table, will aggr table be cleaned?

https://github.com/4paradigm/OpenMLDB/blob/c2d9c8df8036418f2a0acb04c2a693f0715dac36/src/tablet/tablet_impl.cc#L3484-L3506

GetAggregators will get all aggr tables of base table, and truncate it.

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Review by aceforeverd — APPROVED** [review]

**Pull request merged**